docs/source/conf.py

```diff
@@ -9,7 +9,7 @@
 project = 'spark-rapids-ml'
 copyright = '2023, NVIDIA'
```

Comment:
Do we need to update copyright year to 2024? Otherwise looks good.

Reply:
Good point.  Changed throughout in new commit.